app/commands/git/sync_track.rb

```diff
@@ -48,6 +48,8 @@ def call
       # Now that the concepts and exercises have synced successfully,
       # we can set the track's synced git SHA to the HEAD SHA
       track.update!(synced_to_git_sha: head_git_track.commit.oid)
+    rescue StandardError => e
+      Github::Issue::OpenForSyncFailure.(track, e.message, head_git_track.commit.oid)
```

Comment:
Nice! How about passing `e` in rather than `e.message`, as we maybe want the trace etc too.